library/src/rng/generator_type.hpp

```diff
@@ -271,7 +271,7 @@ struct rocrand_generator_impl_base
         return m_stream;
     }
 
-    void set_stream(hipStream_t stream)
+    virtual void set_stream(hipStream_t stream)
```

Comment:
Nice catch, thanks. This was left over from a previous version of the change, sorry about that. I'll push a follow-up commit shortly that fixes up the issues with the poisson distribution, which make it necessary again. When the user calls set_stream() on a subclass of rocrand_generator_impl_base (eg. lfsr113_prng), we need to let the subclass' m_poisson data member know that the stream has changed. We can do that by overriding this function so that it calls m_poisson.set_stream(new_stream).